bool target_found
float32 center_x
float32 center_y
float32 target_x
float32 target_y